### PR TITLE
Cortex bump and dashboard enhancement

### PR DESCRIPTION
**Pull request by jgehrcke**
Also see

https://github.com/cortexproject/cortex/commit/7273defebca2783248a22ae1bc87b9c9abe5d54e
https://github.com/cortexproject/cortex/commit/16265a049d1d7b8498bd54b9ba154acfc98bc025
https://github.com/cortexproject/cortex/commit/9ba848b46c8f0b60302d422bdebf9c5e22f9f301
https://github.com/cortexproject/cortex/commit/ff0d52e6d2a3b1be11c4111604fe5f79c892c967
https://github.com/cortexproject/cortex/commit/a2be3d834ee380fc9364d905e1461dc407b5d6e8

and generally 

https://github.com/cortexproject/cortex/blob/master/CHANGELOG.md#master--unreleased

### CODE REVIEW

**Comment by jgehrcke** [comment]
Screenshot from 2021-02-24 16-27-15

File: packages/controller/src/resources/monitoring/system/dashboards/cortex-overview-per-tenant.json [hunk 1]
```diff
@@ -2374,57 +2219,48 @@
...
       {
-        "allValue": null,
         "current": {
-          "selected": true,
-          "tags": [],
-          "text": "default",
-          "value": "default"
+          "selected": false,
+          "text": "v3",
+          "value": "v3"
```
Comment (triclambert) [inline packages/controller/src/resources/monitoring/system/dashboards/cortex-overview-per-tenant.json:2231]: How about omitting any changes here and I will update with the new dynamic list selection after you merge it?

**Comment by jgehrcke** [comment]
`main-test-gcp` was affected by #277

File: packages/controller/src/resources/monitoring/system/dashboards/cortex-overview-per-tenant.json (hunk 1 above)
Comment (jgehrcke) [reply packages/controller/src/resources/monitoring/system/dashboards/cortex-overview-per-tenant.json:2231]: now that CI passed, I'd love to move on -- feel free to improve this in your PR :)

**Pull request merged**